flycheck.el

```diff
@@ -521,20 +521,43 @@ enabled.  Changing it will not affect buffers which already have
   :package-version '(flycheck . "0.15")
   :safe #'booleanp)
 
+(defconst flycheck-minimum-level-choices
```

Comment:
I'm not sure that a variable is the right way to do this.  Did you check whether we can [define a custom type](https://www.gnu.org/software/emacs/manual/html_node/elisp/Defining-New-Types.html#Defining-New-Types) for this one?  Probably even one that dynamically provides a choice for all defined levels in the current Emacs session?